lhotse/recipes/icsi.py

```diff
@@ -194,14 +194,14 @@ def download_icsi(
     """
     Download ICSI audio and annotations for provided microphone setting.
     :param target_dir: Pathlike, the path in which audio and transcripts dir are created by default.
-    :param audio_dir: Pathlike (default = None), the path to store the audio data.
-    :param transcripts_dir: Pathlike (default = None), path to store the transcripts data
+    :param audio_dir: Pathlike (default = '<target_dir>/audio'), the path to store the audio data.
```

Comment:
```suggestion
    :param audio_dir: Pathlike (default = '<target_dir>/speech'), the path to store the audio data.
```

is that right?

Reply:
Yes, it is. These are set at the very top of the download_icsi function. (line 204 and 205)
The reason I left the argument default `= None` is that these dirs are supposed to be relative to the target dir if they are not specified explicitly. 
Does that make sense?